src/hooks/resources/device.ts

```diff
@@ -576,3 +589,59 @@ async function checkSupervisorReleaseUpgrades(
 		}
 	}
 }
+
+async function setSupervisorReleaseResource(
+	api: sbvrUtils.PinejsClient,
+	deviceIds: number[],
```

Comment:
Since this Fn only handles the first device ID in this list, I would like it to either;
- take a single ID, or
- set the value for all the IDs

Reply:
👍

Reply:
done